hack/bin/generate-nix-overrides.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
We could try to replace this with a script that reads cabal.project instead, right?

Reply:
We could, but parsing cabal.project doesn't seem like fun. If this works well with `shellFor`, we could just get rid of those `source-package-repository` sections from cabal.project.